views.py

```diff
@@ -20,3 +20,21 @@ async def index(request: Request, user: User = Depends(check_user_exists)):
         "nostrmarket/index.html",
         {"request": request, "user": user.dict()},
     )
+
+
+@nostrmarket_ext.get("/market", response_class=HTMLResponse)
+async def market(
+    request: Request,
+    stall_id: str = Query(None),
+    product_id: str = Query(None),
+    merchant_pubkey: str = Query(None),
+):
+    return nostrmarket_renderer().TemplateResponse(
+        "nostrmarket/market.html",
+        {
```

Comment:
- why forwards `stall_id`, `product_id` and `merchant_pubkey` back to the customer
- these values have been received on the request in the first place (therefore the customer already has them)

Reply:
left overs from the initial plan on having query params. fixed